src/main/java/com/example/onlinebookstore/dto/UserLoginRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.onlinebookstore.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserLoginRequestDto {
+    @Email
+    @NotBlank
+    private String email;
+    @NotBlank
+    @Size(min = 8, max = 20)
```

Comment:
I think password could be much bigger that 20 chars

Reply:
done